tests/topotests/bgp_set_aspath_exclude/test_bgp_set_aspath_exclude.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
```

Comment:
Please add SPDX-License tag, and drop all the Copyright below stuff.

Reply:
done